src/commands/workspace/stop.ts

```diff
@@ -39,28 +38,26 @@ export default class Stop extends Command {
   async run() {
     const { flags } = this.parse(Stop)
     const { args } = this.parse(Stop)
-    const ctx: any = {}
 
-    const tasks = new Listr([], { renderer: 'silent' })
+    const workspaceId = args.workspace
 
-    const apiTasks = new ApiTasks()
-    const cheTasks = new CheTasks(flags)
-    const workspaceTasks = new WorkspaceTasks(flags)
+    let cheApiUrl = flags[CHE_API_URL_KEY]
+    if (!cheApiUrl) {
+      const kube = new KubeHelper(flags)
+      if (!await kube.hasReadPermissionsForNamespace(flags.chenamespace)) {
+        throw new Error(`"--${CHE_API_URL_KEY}" argument is required`)
```

Comment:
`Use flag --${CHE_API_URL_KEY} to provide Eclipse Che API endpoint`
pls update all errors like this